src/model/immutable/EditorState.js

```diff
@@ -210,6 +210,10 @@ class EditorState {
     return this.getImmutable().get('inlineStyleOverride');
   }
 
+  static setInlineStyleOverride(editorState: EditorState, inlineStyleOverride: DraftInlineStyle): EditorState {
+    return EditorState.set(editorState, { inlineStyleOverride });
+  }
```

Comment:
Styling nit: we use an 80-char limit on lines, so it would be good to have this as:

```javascript
static setInlineStyleOverride(
  editorState: EditorState,
  inlineStyleOverride: DraftInlineStyle
): EditorState {
  return EditorState.set(editorState, { inlineStyleOverride });
}
```

Sorry for nitpicking on this, it's just that if it's not in the PR, I'll have to do it myself anyway. :)